Createuseraccount

amarachi4423 opened the pull request:
- Added OTP and phone number database
- Implemented phone verification in user account
- Added additional verification checks

amarachi4423 committed:
Create user acccount

Created the flow for user creation, added the phone number, otp verification then entry of user email address, password, and first and last name

amarachi4423 committed:
phonenumber verification

Updated it to be before otp verification

amarachi4423 committed:
new create user

Added the new phone number verification and otp repo

AdepojuToyeeb commented on src/api/users/users.module.ts:
```diff
@@ -1,13 +1,13 @@
+  imports: [TypeOrmModule.forFeature([Users, OtpAuth])],
+  controllers: [UserController],
+  providers: [UserService],
+  exports: [TypeOrmModule],
```
```suggestion

```

amarachi4423 committed:
Update src/api/users/users.module.ts

Co-authored-by: Adepoju Toyeeb <adewale.adepoju14@gmail.com>

amarachi4423 committed:
Createuseraccount

New user creation flow for either phone number or email signup, user completes signup with full name and then password

amarachi4423 committed:
Merge branch 'createuseraccount' of https://github.com/AdepojuToyeeb/safetynet-backend into createuseraccount

AdepojuToyeeb reviewed (approved):
fantastic
